test-programs/compressed-token-test/tests/test.rs

```diff
@@ -432,6 +435,226 @@ async fn test_delegation() {
     kill_prover();
 }
 
+#[tokio::test]
+#[ignore]
```

Comment:
why ignore shouldn't this succeed?

Reply:
It actually doesn't suceed, trying to investigate why.

This assertion fails

```rust
        assert_eq!(
            transfer_recipient_token_compressed_account
                .token_data
                .delegate,
            delegates[i]
        );
```

To be precise, `token_data.delegate` is `None`. I'm trying to check if there is something in the instruction which prevents applying of the correct `token_data` when the amount is zero.

Reply:
nvm, fixed